src/Microdown/MicInlineElement.class.st

```diff
@@ -161,3 +161,8 @@ MicInlineElement >> substring: aSubString [
 MicInlineElement >> text [
  	^ substring
 ]
+
+{ #category : #accessing }
```

Comment:
I'm not sure that this is always true. May be we will add a new superclass to Bold, monospace, Strike and italic and we could move this method there but for now this is ok.